.circleci/config.yml

```diff
@@ -129,7 +129,8 @@ workflows:
             container=community-api,name=FEATURES_APPLY_LIMITED_ACCESS_MARKERS,                       value=true,
             container=community-api,name=FEATURES_NOMS_UPDATE_MULTIPLE_EVENTS_UPDATE_BULK_KEY_DATES,  value=false,
             container=community-api,name=FEATURES_NOMS_UPDATE_MULTIPLE_EVENTS_UPDATE_KEY_DATES,       value=true,
-            container=community-api,name=FEATURES_NOMS_UPDATE_MULTIPLE_EVENTS_UPDATE_PRISON_LOCATION, value=true
+            container=community-api,name=FEATURES_NOMS_UPDATE_MULTIPLE_EVENTS_UPDATE_PRISON_LOCATION, value=true,
+            container=community-api,name=FEATURES_REGISTERS_COURT_CODE_ALLOWED_PATTERN,               value=XXXX[A-Z]{2}
```

Comment:
For now only allow a very restricted number of courts to be updated in test, and none can be updated anywhere else